### PR TITLE
[1LP][RFR]Automate: test_vm_right_size_recommendation_back_button

### PR DESCRIPTION
**Pull request by valaparthvi**
## Purpose or Intent

- __Extending__ 
    1. `RightSizeView`  by adding widgets and `is_displayed` property to it.
- __Adding tests__ 
    1. `test_vm_right_size_recommendation_back_button`
- __Enhancement__ 
    1. Add navigator for `RightSizeView` by removing existing navigator registered against `OpenstackInstance` and writing a common one registered against `VM`.

### PRT Run
{{ pytest: cfme/tests/webui/test_general_ui.py -k "test_vm_right_size_recommendation_back_button" -vvvv }}

### CODE REVIEW

File: cfme/tests/webui/test_general_ui.py [hunk 1]
```diff
@@ -339,6 +339,34 @@ def test_tls_openssl_verify_mode(temp_appliance_preconfig, request):
...
+            1. Navigate to a VM's details page.
+            2. From `Configuration` dropdown, select `Right Size Recommendations`.
+            3. Click on `Back` button.
+        expectedResults:
+            1.
+            2.
+            3. Back button must bring back to the details page.
+    """
+    view = navigate_to(full_template_vm, "RightSize")
+    assert view.back_button.is_displayed
```
Comment (digitronik) [inline cfme/tests/webui/test_general_ui.py]: @valaparthvi steps 3 suggesting to `click` on `back_button` and expected result should assert `Details` Page.

File: cfme/tests/webui/test_general_ui.py [hunk 2]
```diff
@@ -339,6 +339,34 @@ def test_tls_openssl_verify_mode(temp_appliance_preconfig, request):
     assert appliance.advanced_settings["smtp"]["openssl_verify_mode"] == "none"
 
 
+@pytest.mark.tier(1)
+@pytest.mark.meta(coverage=[1733207])
```
Comment (digitronik) [inline cfme/tests/webui/test_general_ui.py]: Q: is it ok to move it to `automate`?

File: cfme/tests/webui/test_general_ui.py (hunk 1 above)
Comment (valaparthvi) [reply cfme/tests/webui/test_general_ui.py]: My bad, I didn't read the testSteps and just wrote the test from what I remembered of the BZ. Made the changes now.

File: cfme/common/vm_views.py [hunk 3]
```diff
@@ -593,11 +593,32 @@ class RightSizeView(BaseLoggedInPage):
     """
     Right Size recommendations page for vms/instances
     """
-    # TODO new table widget for right-size tables
-    # They're H3 headers with the table as following-sibling
 
-    # Only name is displayed
-    is_displayed = displayed_not_implemented
+    title = Text("#explorer_title_text")
```
Comment (mshriver) [inline cfme/common/vm_views.py]: Nice!

**Pull request merged**